src/tests/HDFFileTests.cpp

```diff
@@ -5,9 +5,9 @@
 #include "../helper.h"
 #include "../json.h"
 #include "AddReader.h"
-#include "HDFFileTestHelper.h"
-#include "ev42_synth.h"
-#include "f142_synth.h"
+#include "tests/helpers/HDFFileTestHelper.h"
```

Comment:
It could be argued that the _tests_ directory should be set as an include directory (using `target_include_directories()`) in the relevant CMake file. However, I care only enough for me to write this comment.